### PR TITLE
Feature 74 Build manifest file

### PR DESCRIPTION
**Pull request by genoud**
Integrated plugin sfdx-build (https://www.npmjs.com/package/sfdx-build) into sfpowerkit with minor changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -672,6 +672,44 @@ EXAMPLE
...
+
+  --apiversion=apiversion                          override the api version used for api requests made by this
+                                                   command
+
+  --json                                           format output as json
+
+  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for this command invocation
+
+EXAMPLES
+  $ sfdx build:fetch:xml --targetusername myOrg@example.com -o package.xml
```
Comment (azlam-abdulsalam) [inline README.md]: Check the examples.. it doesnt align with the command name

File: package.json [hunk 2]
```diff
@@ -45,6 +45,7 @@
     "simple-git": "^1.124.0",
     "tslib": "1",
     "unzipper": "0.10.3",
+    "xml-js": "^1.6.11",
```
Comment (azlam-abdulsalam) [inline package.json]: Why are we carrying xml-js and xml2js.. Can we consolidate into one

File: src/impl/metadata/packageBuilder.ts [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+export class Packagexml {
+  public configs: BuildConfig;
+  private conn: Connection;
+
+  constructor(conn: Connection, configs: BuildConfig) {
+    this.conn = conn;
+    this.configs = configs;
+  }
+
+  public async build() {
```
Comment (azlam-abdulsalam) [inline src/impl/metadata/packageBuilder.ts]: Can we break this function to a set of functions, so its readable

File: src/impl/metadata/packageBuilder.ts [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+  };
+}
+
+if (Symbol["asyncIterator"] === undefined) {
+  // tslint:disable-next-line:no-any
+  (Symbol as any)["asyncIterator"] = Symbol.for("asyncIterator");
+}
+
+const STANDARD_VALUE_SETS = [
+  "AccountContactMultiRoles",
```
[the remark this answers is not in the document]
Comment (genoud) [reply src/impl/metadata/packageBuilder.ts]: StandardValueset does not support Wilcard and there is no way to fetch them automatically.  This lis need to be updated a each salesforce release.

File: src/impl/metadata/packageBuilder.ts [hunk 5]
```diff
@@ -0,0 +1,372 @@
+import { Connection } from "@salesforce/core";
+import * as _ from "lodash";
+import * as convert from "xml-js";
+import * as fs from "fs";
+import * as path from "path";
+import FileUtils from "../../utils/fileutils";
+
+declare global {
+  interface Array<T> {
```
[the remark this answers is not in the document]
Comment (genoud) [reply src/impl/metadata/packageBuilder.ts]: Removed, not necessary.

**Pull request merged**